package/MDAnalysis/analysis/hole2/__init__.py

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
add boring boilerplate file header (comments/licence...)

Reply:
Copyright header on new files (gotta get that line count 🙃)